src/thematic/1_water/sfp-105-nl_bemonsteren-peilbuis/12_appendices.Rmd

```diff
@@ -0,0 +1,35 @@
+# (APPENDIX) Bijlagen {.unnumbered}
+
+# Checklist met te noteren informatie in het veld
+
+```{=html}
+<!--
+Oplijsting van de bijlages die toegevoegd zijn aan dit protocol.
+O.a. invulformulier of tabel met projectspecifieke gegevens voor de variabelen (bv afmetingen van proefvlak, diepte van oppervlaktemonster bodem, ...).
+-->
+```
+-   Project
+-   Datum
+-   Monsternemer
+-   Meetpuntcode
+-   StaalID / LaboID
+-   pH
+-   EC (μS/cm)
+-   Opmerkingen
+-   Controlemetingen pH (start & einde)
+
+# AFHANKELIJKHEDEN (Indien van toepassing)
+
+Procedures waarnaar in deze procedure verwezen wordt:
+
+-   SOP-005: richtlijnen voor het conserveren van watermonsters
+
+-   SOP-006: klaarzetten materiaal voor staalname watermonsters
+
+-   SIP-054: Digitale veldmeter pH-EC SIP-xxx: gebruik en onderhoud van een peristaltische pomp
+
+-   SIP-xxx: gebruik en onderhoud van een dompelpomp
+
+Procedures die verwijzen naar deze procedure:
+
+-   Niet van toepassing
```

Comment:
Deze sectie best verwijderen. Indien je naar deze protocols wil verwijzen, dan zal je ofwel naar de bestaande website met INBO protocols moeten verwijzen, ofwel deze protocols op voorhand toevoegen aan protocolsource.  Afhankelijkheden in `01_afhankelijkheden.Rmd` reserveren we enkel en alleen voor afhankelijkheden tussen protocols gepubliceerd via de `protocolsource` repo. Daarbij zijn circulaire verwijzingen niet toegelaten. Je kan dus alleen naar een protocol verwijzen in `01_afhankelijkheden.Rmd` als het protocol al gepubliceerd werd in het nieuwe systeem.

Reply:
De sectie is verwijderd. In de tekst zijn weblinks toegevoegd naar de respectievelijke SIP's en SOP's.